src/staticanalysis/bot/default.nix

```diff
@@ -175,6 +175,7 @@ let
         "SHELLCHECK=${shellcheck}/bin/shellcheck"
         "MOZ_AUTOMATION=1"
         "MOZBUILD_STATE_PATH=/tmp/mozilla-state"
+        "_JAVA_OPTIONS=-Duser.home=\"/tmp/mozilla-state\""
```

Comment:
Nit: the `\"` might not be needed, which would make this more readable: `"_JAVA_OPTIONS=-Duser.home=/tmp/mozilla-state"`.